# Generated for NaoQi method ALMemory.insertDataString
# Allows any program to insert data into ALMemory.

# Name of the data to be inserted.
string name
# Value of the data
string data
---


